pro/src/components/pages/Offers/Offer/OfferDetails/OfferForm/OfferCategories/OfferCategories.jsx

```diff
@@ -71,7 +73,14 @@ const OfferCategories = ({
           )
         }
         updateFormErrors({})
-        setSubCategoriesOptions(buildSelectOptions('id', 'proLabel', options))
+        setSubCategoriesOptions(
+          options
+            .map(item => ({
+              id: item['id'].toString(),
+              displayName: item['proLabel'],
+            }))
+            .sort((a, b) => a.displayName.localeCompare(b.displayName, 'fr'))
+        )
```

Comment:
Pour ce tri il y a peut être un utils à sortir pour le coup ?

```suggestion
        setSubCategoriesOptions(
          sortByDisplayName(
            options
              .map(item => ({
                id: item['id'].toString(),
                displayName: item['proLabel'],
              }))
          )
        )
```

Reply:
Je ne comprend pas pourquoi on s’autoriserait une fonction passe plat vers `sort() / localeCompare()` et pas une fonction plus complète qui répond au besoin des composant : `layout/input/Select` ou `ui-kit/form/Select`

Reply:
Franchement différent : 
sort / localCompare = répétion stricte du même code, action unique (trier un tableau sans rien modifier), toujours la même interface en entrée comme en sortie (données du Select)
buildOptions  = mélange de problématiques techniques et métier, interfaces différentes selon les cas, helper magique

Reply:
Je suis en total désaccord, mais je peut faire avec.